tracing/src/test/java/com/palantir/tracing/TracersTest.java

```diff
@@ -203,7 +203,7 @@ public void testWrapCallable_callableTraceIsIsolated() throws Exception {
     public void testWrapCallable_traceStateIsCapturedAtConstructionTime() throws Exception {
         Tracer.startSpan("before-construction");
         Callable<Void> callable = Tracers.wrap(() -> {
-            assertThat(Tracer.completeSpan().get().getOperation()).isEqualTo("before-construction");
+            assertThat(Tracer.completeSpan().get().getOperation()).isEqualTo("deferred");
```

Comment:
Instead of the string `deferred`, what do you think about using `DeferredTracer(unnamed operation)`.  I hope this would be enough information for people to hunt down usages of 'DeferredTracer' in their codebase and hopefully be prompted to give them an operation name?